<ui version="4.0" >
 <class>OmgLayerSetManagerBase</class>
 <widget class="QDialog" name="OmgLayerSetManagerBase" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Layerset manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/om_icon_small.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="grpLayerset" >
     <property name="title" >
      <string>Available layersets</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="6" >
       <widget class="QTableWidget" name="tblLayerSets" >
        <property name="whatsThis" >
         <string>Table of the currently available layersets</string>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text" >
          <string>FileName</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QToolButton" name="toolNew" >
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/resources.qrc" >:/filenew.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QToolButton" name="toolCopy" >
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/resources.qrc" >:/editcopy.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QToolButton" name="toolDelete" >
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../resources/resources.qrc" >:/edittrash.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="pbnImport" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="pbnExport" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>71</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="grpLayers" >
     <property name="title" >
      <string>Details</string>
     </property>
     <layout class="QGridLayout" >
      <item row="3" column="0" colspan="2" >
       <widget class="QListWidget" name="lstLayers" >
        <property name="toolTip" >
         <string>Layers assigned to current layerset</string>
        </property>
        <property name="whatsThis" >
         <string>List of the layers assigned to the current layerset</string>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Mask layer</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QComboBox" name="cboMask" >
          <property name="whatsThis" >
           <string>Layer that is to be used as a mask for the current layerset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolNewMaskLayer" >
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/fileopen.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leName" >
        <property name="whatsThis" >
         <string>Short name for the current layerset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leDescription" >
        <property name="whatsThis" >
         <string>Description of the current layerset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="lblLayers" >
          <property name="text" >
           <string>Layers</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolAddLayers" >
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/edit_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolDeleteLayers" >
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/edit_remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>341</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pbnClose" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pbnApply" >
     <property name="whatsThis" >
      <string>Apply the changes that have just been made to these layerset details</string>
     </property>
     <property name="text" >
      <string>Apply</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbnClose</sender>
   <signal>clicked()</signal>
   <receiver>OmgLayerSetManagerBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>510</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel" >
     <x>164</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
